Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -0,0 +1,1586 @@
+import demistomock as demisto
+# Common functions script
+# =======================
+# This script will be appended to each server script before being executed.
+# Please notice that to add custom common code, add it to the CommonServerUserPython script
+from datetime import datetime, timedelta
+import time
+import json
+import sys
+import os
+import re
+from collections import OrderedDict
+
+import xml.etree.cElementTree as ET
+
+IS_PY3 = sys.version_info[0] == 3
+# pylint: disable=undefined-variable
+if IS_PY3:
+    STRING_TYPES = (str, bytes)  # type: ignore
+    STRING_OBJ_TYPES = (str)
```

Comment:
in order for the type be tuple (consistent with the else part)
```suggestion
    STRING_OBJ_TYPES = (str, )
```